set.mm

```diff
@@ -411118,22 +411123,22 @@ a loop (see ~ uspgr1v1eop ) is a singleton of a singleton.  (Contributed
   $}
 
   ${
-    $d G e k n v $.  $d G V v $.  $d e f i k n v w $.  $d e f k n p q v w y $.
+    $d G e k n v $.  $d V v $.  $d e f i k n v w $.  $d e f k n p q v w y $.
     sumvtxdg2size.v $e |- V = ( Vtx ` G ) $.
     sumvtxdg2size.i $e |- I = ( iEdg ` G ) $.
     sumvtxdg2size.d $e |- D = ( VtxDeg ` G ) $.
     $( The sum of the degrees of all vertices of a finite pseudograph of finite
        size is twice the size of the pseudograph.  See equation (1) in section
        I.1 in [Bollobas] p. 4.  Here, the "proof" is simply the statement
        "Since each edge has two endvertices, the sum of the degrees is exactly
-       twice the number of edges".  Although the proof for a simple graph would
-       be much shorter, the formal proof of this theorem (for pseudographs) is
-       much more complicated, taking also the used auxiliary theorems into
-       account.  Although this theorem would hold also for infinite
-       pseudographs and pseudographs of infinite size, the proof of this most
-       general version (see theorem "sumvtxdg2size" below) would require much
-       more auxiliary theorems (e.g. the extension of the sum ` sum_ ` over an
-       arbitrary set).
+       twice the number of edges".  The formal proof of this theorem (for
+       pseudographs) is much more complicated, taking also the used auxiliary
+       theorems into account.  The proof for a (finite) simple graph (see
+       ~ usgr1th ) would be shorter, but nevertheless still laborious.
+       Although this theorem would hold also for infinite pseudographs and
+       pseudographs of infinite size, the proof of this most general version
+       (see theorem "sumvtxdg2size" below) would require much more auxiliary
```

Comment:
I'll replace "much" by "many"..
I'll also add a comma after e.g., because it is American English (in British English it is not used), and we decided to use American English in set.mm.